pkg/schema/v1/job.go

```diff
@@ -166,6 +171,28 @@ func (j *Job) Obtain(k8s kmetav1.Object) {
 	j.Yaml = string(output)
 }
 
+func (j *Job) getIcingaState(job *kbatchv1.Job) (IcingaState, string) {
+	if gracePeriodReason := IsWithinGracePeriod(j); gracePeriodReason != nil {
```

Comment:
There is no need for a grace period as we will do the following:
* If condition is complete, return ok: Job .. has finished its execution successfully with %d necessary pod completions.
* If condition is failed, return critical. Message should stay the same.
* If condition is suspend, return unknown. Message should indicate that the job is suspended.
* If none of the above applies, return pending. Job .. is running since %s with currently %d active, %d completed and %d failed pods. Successful termination requires %d pod completions. The back-off limit is %d.
* If ActiveDeadlineSeconds is set, the pending message should be extended: Deadline for completion is %s.